### PR TITLE
Russia: Name swapping and new placetypes for admin1

### LINKED ISSUE #1397: Update Russia admin1 names
Records for first division admin in Russia have a mix of Cyrillic and English `wof:name` values (examples [1](https://spelunker.whosonfirst.org/id/85688167/), [2](https://spelunker.whosonfirst.org/id/85688189/)). Some of the `wof:name` values have additional trailing characters ([example](https://spelunker.whosonfirst.org/id/85687985/)).

In addition, the `name:` properties can be updated to include more translation variants. We can also add English and localized placetypes (okrug, federal city, etc) to the records.

See:
- https://en.wikipedia.org/wiki/Federal_subjects_of_Russia
- http://statoids.com/uru.html

### PR DESCRIPTION
**Pull request by stepps00**
Fixes #1397.

Adds new `name:eng_x_preferred` and `name:eng_x_variant` name translations, as well as new preferred English and localized placetypes. Also includes updated `wof:name` values in some cases.

### CODE REVIEW

**Comment by nvkelso** [comment]
Generally looks good, but please also add `label:eng_x_preferred_longname` support?

- https://github.com/whosonfirst/whosonfirst-properties/blob/master/properties/label/README.md#lang_x_preferred_longname

**Comment by stepps00** [comment]
https://github.com/whosonfirst-data/whosonfirst-data/pull/1396#issuecomment-441913280 addressed in https://github.com/whosonfirst-data/whosonfirst-data/pull/1396/commits/4f7f06172c99f2de2923860d49e7825d7558f4c6.

File: data/856/880/85/85688085.geojson [hunk 1]
```diff
@@ -925,12 +937,11 @@
         "lbe",
         "koi"
     ],
-    "wof:lastmodified":1537304039,
+    "wof:lastmodified":1543351549,
     "wof:name":"Moscow City",
```
Comment (nvkelso) [inline data/856/880/85/85688085.geojson]: Generally we'd drop the ` City` part and add whatever "region" to the region's name.

File: data/856/880/85/85688085.geojson [hunk 2]
```diff
@@ -185,10 +194,13 @@
         "M\u00e5ssca"
     ],
     "name:eng_x_preferred":[
-        "Moscow"
+        "Moscow City"
```
Comment (nvkelso) [inline data/856/880/85/85688085.geojson]: Generally we'd drop the ` City` part and add whatever "region" to the region's name.

File: data/856/880/85/85688085.geojson (hunk 1 above)
Comment (nvkelso) [reply data/856/880/85/85688085.geojson]: Please make sure that's correct for the 2 Moscow features here.

File: data/856/880/69/85688069.geojson [hunk 3]
```diff
@@ -11,6 +11,15 @@
     "geom:longitude":30.213247,
     "iso:country":"RU",
     "label:eng_x_abbreviation":"SP",
+    "label:eng_x_preferred_longname":[
+        "Saint Petersburg"
```
Comment (nvkelso) [inline data/856/880/69/85688069.geojson]: See prior comments about "city" and Moscow.

**Pull request merged**